src/ui/ProcessManager.js

```diff
@@ -25,12 +25,28 @@ export default class ProcessManager extends React.Component {
     return pids.indexOf(this.state.selectedPid) !== -1;
   }
 
+  canDebug() {
+    if (!this.state.selectedPid) return false;
+    const pids = this.state.processData.map(p => p.pid);
+
+    // verify that select pid is in list of processes
+    //return pids.indexOf(this.state.selectedPid) !== -1;
+
+    return true;
```

Comment:
The line above is eventually not needed? or it's just for debugging?
Probably worth factorizing with `canKill` :)

Reply:
Done